code/modules/mob/living/carbon/human/life.dm

```diff
@@ -28,25 +28,26 @@
 	if (QDELETED(src))
 		return 0
 
-	if(.) //not dead
-		handle_active_genes()
+	if(!IsInStasis())
+		if(.) //not dead
+			handle_active_genes()
 
-	if(stat != DEAD)
-		//heart attack stuff
-		handle_heart()
+		if(stat != DEAD)
+			//heart attack stuff
+			handle_heart()
```

Comment:
This can kill you if you are having a heart attack.
https://github.com/tgstation/tgstation/blob/5b89dea70f84fe72a653c4d65fbf056206764fc3/code/modules/mob/living/carbon/human/life.dm#L322-L326